lib/chef/knife/bmcs_server_list.rb

```diff
@@ -25,10 +26,39 @@ def run
         options = {}
         options[:limit] = config[:limit] if config[:limit]
 
+        servers = {}
+
         response = compute_client.list_instances(compartment_id, options)
 
-        display_list(response, ['Display Name', 'ID', 'State']) do |item|
-          [item.display_name, item.id, item.lifecycle_state]
+        if config[:format] == 'summary' && response.data
+          # build a hash of hashs, keyed off of server_id
+          vnics = compute_client.list_vnic_attachments(compartment_id, {})
```

Comment:
We might have paging issue here, since 1) the same limit is not applied to this list query as in list_instances, and 2) even if we did, there may be multiple vinc-attachments per instance. So, I would recommend calling this once per instance with instance_id optional param set.